src/bugsnag.js

```diff
@@ -261,17 +261,16 @@
   var _hasAddEventListener = (typeof window.addEventListener !== "undefined");
 
   // Setup breadcrumbs for click events
-  function trackClicks() {
-    if(!getBreadcrumbSetting("autoBreadcrumbsClicks", true)) {
-      return;
-    }
-
+  function setupClickBreadcrumbs() {
     if (!_hasAddEventListener) {
       return;
     }
 
-
     var callback = function(event) {
+      if(!getBreadcrumbSetting("autoBreadcrumbsClicks", true)) {
```

Comment:
I know this is just moving what we had before, but the second argument here doesn't do anything and it might be nice to take it out to avoid confusion.